### PR TITLE
[MM-742]: Added a note about the label of pr/issue if the subscription is specific to a label

### PR DESCRIPTION
**Pull request by Kshitij-Katiyar**
### Summary
Added a note to the subscription message post about the label of pr/issue if the subscription is specific to a label

### Screenshots
Screenshot from 2024-09-25 17-58-16
Screenshot from 2024-09-25 17-58-25

### How to Test
- Setup the plugin
- Create a subscription specific to a label
- Check the subscription post after creating a PR/Issue with the same label

### CODE REVIEW

File: server/plugin/template.go [hunk 1]
```diff
@@ -204,11 +210,11 @@ Assignees: {{range $i, $el := .Assignees -}} {{- if $i}}, {{end}}{{template "use
 
 	template.Must(masterTemplate.New("newPR").Funcs(funcMap).Parse(`
 {{ if eq .Config.Style "collapsed" -}}
-{{template "repo" .Event.GetRepo}} New pull request {{template "pullRequest" .Event.GetPullRequest}} was opened by {{template "user" .Event.GetSender}}.
+{{template "repo" .Event.GetRepo}} New pull request {{template "pullRequest" .Event.GetPullRequest}} was opened by {{template "user" .Event.GetSender}}{{template "subscriptionLabel" (dict "Label" .Label)}}.
```
Comment (raghavaggarwal2308) [inline server/plugin/template.go]:
```suggestion
{{template "repo" .Event.GetRepo}} New pull request {{template "pullRequest" .Event.GetPullRequest}} was opened by {{template "user" .Event.GetSender}}{{template "subscriptionLabel" .}}.
```
Do we need to pass it as a dict?

**Review by raghavaggarwal2308 — CHANGES_REQUESTED** [review]

File: server/plugin/template.go (hunk 1 above)
Comment (Kshitij-Katiyar) [reply server/plugin/template.go]: @raghavaggarwal2308 
I believe we don't need to pass the entire context to the `subscriptionLabel` template. Passing just the label using the `dict` should be sufficient.

Comment (raghavaggarwal2308) [reply server/plugin/template.go]: We can just send .Label for here and access it as {{ . }}

File: server/plugin/template.go [hunk 2]
```diff
@@ -190,6 +190,12 @@ func init() {
 {{- if .Labels }}
 Labels: {{range $i, $el := .Labels -}}` + "{{- if $i}}, {{end}}[`{{ $el.Name }}`]({{ $.RepositoryURL }}/labels/{{ $el.Name | pathEscape }})" + `{{end -}}
 {{ end -}}
+`))
+
+	template.Must(masterTemplate.New("subscriptionLabel").Funcs(funcMap).Parse(`
+{{- if .Label }}
```
Comment (raghavaggarwal2308) [inline server/plugin/template.go]:
```suggestion
{{- if . }}
```

**Review by raghavaggarwal2308 — APPROVED** [review]

**Comment by Kshitij-Katiyar** [comment]
Created a similar PR on MM with all these changes hence closing this one

**Pull request closed without merging**